app/Models/MapMarker.php

```diff
@@ -318,7 +321,7 @@ protected function popup(): string|null
                 ' . $body . '
             </div>`)
             .on(`mouseover`, function (ev) {
```

Comment:
Feel like having an if around the whole `on()` would be cleaner, rather than having an empty `mouseover` :)